src/targeting/build-page-targeting-consentless.ts

```diff
@@ -0,0 +1,71 @@
+import type { Participations } from '@guardian/ab-core';
+import type { ConsentState } from '@guardian/consent-management-platform/dist/types';
+import type { CountryCode } from '@guardian/libs';
+import type { PageTargeting } from './build-page-targeting';
+import { buildPageTargeting } from './build-page-targeting';
+
+const consentlessTargetingKeys = [
+	'ab',
+	'at',
+	'bl',
+	'bp',
+	'br',
+	'cc',
+	'ct',
+	'dcre',
+	'edition',
+	'k',
+	'rp',
+	's',
+	'se',
+	'sens',
+	'sh',
+	'si',
+	'skinsize',
+	'su',
+	'tn',
+	'url',
+	'urlkw',
+] as const;
+
+type ConsentlessTargetingKeys = typeof consentlessTargetingKeys[number];
+
+type ConsentlessPageTargeting = Partial<
+	Pick<PageTargeting, ConsentlessTargetingKeys>
+>;
+
+const isConsentlessKey = (key: unknown): key is ConsentlessTargetingKeys => {
+	return consentlessTargetingKeys.includes(key as ConsentlessTargetingKeys);
```

Comment:
This is just code golf but this could be an implicit return

Reply:
Good call

Reply:
🏌️